testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/composites/CompositeRoleTest.java

```diff
@@ -56,6 +56,8 @@ public void addTestRealms(List<RealmRepresentation> testRealms) {
                 .ssoSessionIdleTimeout(3000)
                 .accessTokenLifespan(10000)
                 .ssoSessionMaxLifespan(10000)
+                .ssoSessionMaxLifespanRememberMe(10000)
```

Comment:
Shouldn't be required to set this in existing tests. This test has nothing to do with your changes.